storagestate/storagestate.go

```diff
@@ -383,13 +383,10 @@ func (storageState *StorageState) StateAcceptance(updateState cloudprotocol.Stat
 	return nil
 }
 
-func (storageState *StorageState) GetInstanceCheckSum(instanceIdent aostypes.InstanceIdent) (string, error) {
-	state, ok := storageState.statesMap[instanceIdent]
-	if !ok {
-		return "", ErrNotFound
-	}
+func (storageState *StorageState) GetInstanceCheckSum(instanceIdent aostypes.InstanceIdent) string {
+	state := storageState.statesMap[instanceIdent]
```

Comment:
But if there is no state for this instance, should we return error?

Reply:
it is not ana error because instance could doesn't;t have ana state